### PR TITLE
Origin/editorialreview cloudconsole

### PR DESCRIPTION
**Pull request by christinaausley**
Editorial review of all Cloud Console docs for grammar, punctuation, and organization. A few comments made inline. Looking forward to your feedback!

### CODE REVIEW

File: docs/product-manuals/cloud-console/manage-clusters/create-cluster.md [hunk 1]
```diff
@@ -3,8 +3,10 @@ id: create-cluster
...
 ---
 
-If you haven't created a cluster yet, the overview will be empty.
+If you haven't created a cluster yet, the **Clusters** page will be empty.
 
 import CreateCluster from './create-cluster-include.md'
 
 <CreateCluster/>
+
+[//]:# (Seems as though this is the method of calling something like a React component into a Markdown file. Should this have more instruction or should I directly link the file in place of the lines above?)
```
Comment (urbanisierung) [inline docs/product-manuals/cloud-console/manage-clusters/create-cluster.md:12]: @christinaausley The snippet above is used to include another Markdown file into this one. On this page everything from the `create-cluster-include.md` file will be included. So nothing else to do here :)

File: docs/product-manuals/cloud-console/manage-clusters/manage-api-clients.md [hunk 2]
```diff
@@ -3,37 +3,44 @@ id: manage-api-clients
...
+The downloaded file contains all necessary information to communicate with your Zeebe instance in the future:
+
+- `ZEEBE_ADDRESS`: Address where your cluster can be reached.
+- `ZEEBE_CLIENT_ID` and `ZEEBE_CLIENT_SECRET`: Credentials to request a new access token.
+- `ZEEBE_AUTHORIZATION_SERVER_URL`: A new token can be requested at this address, using the
 
-- `ZEEBE_ADDRESS`: address where your cluster can be reached.
-- `ZEEBE_CLIENT_ID` and `ZEEBE_CLIENT_SECRET`: credentials to request a new access token.
-- `ZEEBE_AUTHORIZATION_SERVER_URL`: a new token can be requested at this address, using the
+[//]:# (Looks like this doc cuts off. How should we finish the sentence?)
```
Comment (urbanisierung) [inline docs/product-manuals/cloud-console/manage-clusters/manage-api-clients.md:46]: Oops ;) We can just cut off before the comma: `A new token can be requested at this address.`

File: docs/product-manuals/cloud-console/manage-organization/manage-users.md [hunk 3]
```diff
@@ -29,6 +29,8 @@ The following table illustrates the rights of each role:
...
 | Users: Manage admins         | X     |       |        |
 
-Users are invited to a Camunda Cloud organization via their e-mail address, which must be accepted by the user. As long as the invitation has not been accepted, the user remains in the 'Pending' state.
+Users are invited to a Camunda Cloud organization via their email address, which must be accepted by the user. As long as the invitation has not been accepted, the user remains in the `Pending` state.
 
-People can also be invited to an organization that does not yet have a Camunda Cloud account. In this case the invited person must first create a Camunda Cloud account and can then gain access to the organization.
+People can also be invited to an organization that does not yet have a Camunda Cloud account. In this case, the invited individual must first create a Camunda Cloud account, and can then gain access to the organization.
+
+[//]:# (Are we trying to say above that people can be invited to an organization with a Camunda Cloud account even if the invited person doesn't have an account? If this is the case, I'd say "People who do not yet have a Camunda Cloud account can also be invited to an organization.")
```
Comment (urbanisierung) [inline docs/product-manuals/cloud-console/manage-organization/manage-users.md:36]: Exactly, your proposal is much better: `People who do not yet have a Camunda Cloud account can also be invited to an organization.`

File: docs/product-manuals/cloud-console/troubleshooting/common-pitfalls.md [hunk 4]
```diff
@@ -1,17 +1,21 @@
...
-- Check if your cluster is _Healthy_: it can happen that a Zeebe cluster is temporarily unavailable. Check the status of your cluster via the cluster overview.
+- Check if your [API Client](../manage-clusters/manage-api-clients.md) has the necessary rights. To interact with Zeebe, the **Scope** `Zeebe` must be set.
+- Check if your credentials are configured correctly. There is a CLI tool that allows you to easily check the status: [`zbctl`](https://www.npmjs.com/package/zbctl). With the command `zbctl status`, you can read the topology. If this command works, the connection can be established.
+- Check if your cluster is **Healthy**: A Zeebe cluster may be temporarily unavailable. Check the status of your cluster via the cluster overview.
+
+[//]:# (Is the note above referring to clicking on the cluster to view its details? If so, I might add more clarification here as to how users can find the cluster overview.)
```
Comment (urbanisierung) [inline docs/product-manuals/cloud-console/troubleshooting/common-pitfalls.md:21]: The cluster status can be observed in both views: the cluster list and the cluster details. A user can see if a cluster is healthy by navigating to the cluster list, so there is no need to go to the cluster details. But: the cluster details give the user a more fine grained view of the status over all products (zeebe, operate, tasklist, optimize).

File: docs/product-manuals/cloud-console/troubleshooting/feedback-and-support.md [hunk 5]
```diff
@@ -3,8 +3,10 @@ id: feedback-and-support
 title: Feedback and support
 ---
 
-If you have any problems, questions or suggestions you can contact us via the Cloud Console Feedback Form.
+If you have any problems, questions, or suggestions, contact us via the Cloud Console Feedback Form.
 
-**Feebback and support** can be submitted or requested via the corresponding entry in the navigation menu. If you have a **license agreement** with us you will be redirected to the [Support queue](https://jira.camunda.com/projects/SUPPORT/) at Camunda. In the Free Plan, contact can be made using the internal form:
+[//]:# (Can we provide a link to the feedback form here, or is it only accessed within Cloud Console itself via the Feedback and Support tab here?)
```
Comment (urbanisierung) [inline docs/product-manuals/cloud-console/troubleshooting/feedback-and-support.md:8]: In the beginning it was a built-in form. Now we just link to the Forum: https://forum.camunda.io/ - so we can use the forum link here.

File: docs/product-manuals/cloud-console/manage-clusters/create-cluster.md (hunk 1 above)
Comment (christinaausley) [reply docs/product-manuals/cloud-console/manage-clusters/create-cluster.md:12]: Great, thanks for confirming!

File: docs/product-manuals/cloud-console/manage-organization/manage-users.md (hunk 3 above)
Comment (christinaausley) [reply docs/product-manuals/cloud-console/manage-organization/manage-users.md:36]: Awesome -- thanks for clarifying.

File: docs/product-manuals/cloud-console/troubleshooting/common-pitfalls.md (hunk 4 above)
Comment (christinaausley) [reply docs/product-manuals/cloud-console/troubleshooting/common-pitfalls.md:21]: How's the following: "Check if your cluster is **Healthy**: A Zeebe cluster may be temporarily unavailable. To check if your cluster is healthy, navigate to the cluster list. Click on the cluster to view its details for a closer view of the cluster status over all products (Zeebe, Operate, Tasklist, Optimize).

File: docs/product-manuals/cloud-console/manage-plan/available-plans.md [hunk 6]
```diff
@@ -3,6 +3,12 @@ id: available-plans
...
+Camunda Cloud was released to the world as Beta in September 2019. To sign up for Camunda Cloud, visit [the sign up page](https://accounts.cloud.camunda.io/signup).
 
-There are three plans available for Camunda Cloud: Free Trial, Professional, and Enterprise. For more information on Camunda Cloud plans, please visit the Camunda Cloud [product page](https://camunda.com/products/cloud/).
+There are three plans available for Camunda Cloud:
+
+- Free Trial
+- Professional
+- Enterprise
+
+For more information on Camunda Cloud plans, visit the Camunda Cloud [product page](https://camunda.com/products/cloud/).
```
Comment (akeller) [inline docs/product-manuals/cloud-console/manage-plan/available-plans.md:14]: @christinaausley can you reach out to Dan Levy about this section? I'd like to get his thoughts on a few things

* Should we change the reference to the GA release date? 
* When we talk about our plans, do we really have 3? Based on the [product page](https://camunda.com/products/cloud/) it looks like you can trial the Professional or Enterprise plans.

**Review by akeller — COMMENTED** [review]
This looks good to me! I left one comment about positioning our plans.

File: docs/product-manuals/cloud-console/manage-plan/available-plans.md (hunk 6 above)
Comment (christinaausley) [reply docs/product-manuals/cloud-console/manage-plan/available-plans.md:14]: @akeller Dan says 1.0 launched in May 2021; 1.1 in July. He also thinks two plans are better suited here. Essentially, two plans with a free trial available. I can adjust these changes on my end.

File: docs/product-manuals/cloud-console/troubleshooting/common-pitfalls.md (hunk 4 above)
Comment (urbanisierung) [reply docs/product-manuals/cloud-console/troubleshooting/common-pitfalls.md]: I like it!

**Comment by christinaausley** [comment]
@urbanisierung Would you like to wait until Schuppi is back before we merge? Definitely no rush, but happy to close this one out if you want to.

**Comment by urbanisierung** [comment]
> @urbanisierung Would you like to wait until Schuppi is back before we merge? Definitely no rush, but happy to close this one out if you want to.

@christinaausley We can merge it as soon as we're ready. From my point of view this can be merged.

**Review by urbanisierung — APPROVED** [review]
/lgtm

thanks!

**Pull request merged**